test/functional/feature_config_args.py

```diff
@@ -83,13 +83,17 @@ def test_log_buffer(self):
         self.stop_node(0)
 
     def test_args_log(self):
-        with self.nodes[0].assert_debug_log(expected_msgs=[
-                'command-line arg: -addnode=some.node',
-                'config-file arg:  -regtest=1',
-        ]):
+        with self.nodes[0].assert_debug_log(
+                expected_msgs=[
+                    'command-line arg: -addnode=some.node',
+                    'config-file arg:  -regtest=1',
+                ],
+                unexpected_msgs=[
+                    'torpassword=torpassword',
+                ]):
             self.start_node(0, extra_args=[
                 '-addnode=some.node',
-                '-torpassword=mypass',
+                '-torpassword=torpassword',
             ])
```

Comment:
This test was not verifying that arguments that should not be logged to `debug.log` are indeed not logged. I wanted to add test the full list of what should not be logged (torpassword, rpcauth, rpcbind, rpcpassword, rpcuser), but specifying any of those except `torpassword` on the command line causes the test framework to fail ("`Error: no RPC connection`), so I couldn't think of a way to test that the logging of those is suppressed. I think this test is still useful, however, because it's pretty easy to see in `init.cpp` that `torpassword` is treated the same as the others (namely, that they all include `ArgsManager::DONT_LOG`).

Reply:
Thanks @LarryRuane for taking the suggestion to cover all the `DONT_LOG` cases and your change to pass only the values to unexpected_messages makes it even better. The only thing I regret is having suggested "satoshi" as the `rpcuser` because it pollutes git grepping; a name that isn't already used by the codebase like "charlie" or "abc123", etc., would have been better.